OlxProject/Tools/driver.rb

```diff
@@ -1,13 +1,16 @@
 require 'rubygems'
 require 'selenium-webdriver'
 
-public def StartTest
+public def start_test
   $driver = Selenium::WebDriver.for :chrome
-  $driver.manage.timeouts.implicit_wait = 5
+
+  $wait = Selenium::WebDriver::Wait.new(:timeout => 10)
+
+  $driver.manage.timeouts.implicit_wait = 10
   $driver.manage.window.maximize
   $driver.navigate.to "https://www.olx.ua/"
 end
 
-public def DriverClose
+public def driver_close
   $driver.quit
 end
```

Comment:
also You might want to use memoization for driver
https://thoughtbot.com/blog/ruby-memoization-and-alternatives